test-support/src/main/java/org/springframework/ldap/test/EmbeddedLdapServer.java

```diff
@@ -85,7 +104,50 @@ public static EmbeddedLdapServer newEmbeddedServer(String defaultPartitionName,
     public void shutdown() throws Exception {
         ldapServer.stop();
         directoryService.shutdown();
-
-        FileUtils.deleteDirectory(workingDirectory);
     }
+
+	private static void initSchemaPartition(DefaultDirectoryService directoryService)
```

Comment:
So they made a non-passive change? If so, we might need to include this in a non point release